runtime/kernel/operator_registry.cpp

```diff
@@ -14,15 +14,74 @@
 
 #include <executorch/runtime/platform/assert.h>
 
-namespace torch {
-namespace executor {
+#ifdef _WIN32
+#include <memory>
+#include <windows.h>
+#include <tchar.h>
+
+#define SHARED_MEMORY_NAME "torch_executor_operator_registry"
+static std::shared_ptr<torch::executor::OperatorRegistry> operator_reg;
```

Comment:
cc @larryliu0820 on the operator changes

Reply:
@hans00 I'd love to understand a bit more about the design here for operator registry (same for backend registry). The way it works now is that we depend on static initializer to collect all ops/backends and put them into the registry. I imagine that should still be the case for Windows? Can you please provide some clarity on why doing things differently?

Reply:
I found shared section could help, not need mmap